lib/Net/SAML2/SP.pm

```diff
@@ -270,6 +284,20 @@ around BUILDARGS => sub {
     return $self->$orig(%args);
 };
 
+sub _build_id {
+    my $self = shift;
+
+    # This allows current clients to override the builder without changing
+    # their code
+    if (my $f = $self->can('generate_sp_desciptor_id')) {
```

Comment:
s/generate_sp_desciptor_id/generate_sp_descriptor_id/

Reply:
spelling mistake in original I assume we do not need to keep it

Reply:
The original function has a spelling mistake. I want to keep it for a couple of releases and than remove it completely.

Reply:
sounds good